src/enums/DatatypeEnum.php

```diff
@@ -0,0 +1,117 @@
+<?php
+namespace ParseCsv\enums;
+
+/**
+ * Class DatatypeEnum
+ *
+ * @package ParseCsv\enums
+ *
+ * todo: needs a basic parent enum class for error handling.
+ */
+class DatatypeEnum
+{
```

Comment:
The curly bracket should go to the line above to match the style of the rest of the project. I fixed it in https://github.com/itexia/parsecsv-for-php/pull/6.